tensorboard/components/tf_backend/tf-backend.ts

```diff
@@ -0,0 +1,28 @@
+/* Copyright 2019 The TensorFlow Authors. All Rights Reserved.
+
+Licensed under the Apache License, Version 2.0 (the 'License');
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an 'AS IS' BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+==============================================================================*/
+
+export * from "../tf-imports/polymer.html";
+export * from "../tf-imports/lodash.html";
+export * from "../vz-sorting/vz-sorting.html";
+export * from "requestManager";
+export * from "urlPathHelpers";
+export * from "router";
```

Comment:
please fix this file, too.

Reply:
Sorry, I forgot about that.  I've renamed it to `index.ts`, removed the non-backend exports, and made the module names relative paths.